topics/transcriptomics/tutorials/scrna-case_monocle3-trajectories/slides.html

```diff
@@ -0,0 +1,463 @@
+[Deconinck et al., 2021](https://doi.org/10.1016/j.coisb.2021.05.005)
+
+---
+### Why different TI methods are designed to infer different kinds of biological processes?
+--
+
+Look at the pipeline presented by [Robrecht Cannoodt et al., 2016](https://onlinelibrary.wiley.com/doi/10.1002/eji.201646347)
+
+--
+
+.image-100[ ![pipeline](../../images/scrna-casestudy-monocle/7_methods.jpg) ]        
+--
+
+There are multiple methods using particular algorithms, or even their combinations, so you must consider which one would be best for analysing your sample.
+
+---
+
+### When analysing your data, consider the following: 
+--
+
+.pull-left[  
+  
+
+- Tissue from which the cells come from   
+
+- Branching points  
+
+- Supervised and unsupervised learning  
+
+- Format of the data  
+
+- Number of cells and features  
+
+- Computing power & running time   
+
+]
+
+--
+
+.pull-right[
+    
+ 
+To help you evaluate which method would work best for your data, check out this [**awesome comparision site - dynguidelines**](https://zouter.shinyapps.io/server/), a part of a larger set of open packages for doing and interpreting trajectories called [the dynverse](https://github.com/dynverse/dynverse).     
+  
+![zouter](../../images/scrna-casestudy-monocle/zouter.jpg) 
+
+]
+
+---
+
+### When analysing your data, consider the following: **Tissue from which the cells come from**
+--
+
+.pull-left[
+    
+  
+  
+- Gives an idea about the topology you expect and the priors you can feed the method with, as well as expected cell types and potential contaminants  
+
+- You need to know about 75% of your data and verify that your analysis shows that, before you can then identify the 25% new information  
+
+- Trajectory analysis is quite a sensitive method, so always check if the obtained computational results make biological sense!
+
+]
+
+--
+
+.pull-right[  
+  
+
+![scheme](../../images/scrna-casestudy-monocle/8_scheme.png)  
+
+<sub><sub><sub> Reprinted from “Heterogeneity of Murine Lymph Node Stromal Cell Subsets”, by BioRender.com (2022) </sub></sub></sub>
+]
+
+---
+### When analysing your data, consider the following: **Branching points**
+
+--
+
+Cells can differentiate or develop in various way, so they may exhibit different topologies. 
+
+--
+
+![table](../../images/scrna-casestudy-monocle/table.jpg)  
+
+---
+
+### When analysing your data, consider the following: **Branching points**  
+  
+  
+  
+.pull-left[
+<br />
+<br />
+- diffusion pseudotime (DPT) and branch points
+ - DPT does not rely on dimension reduction and can thus detect subtle changes in high-dimensional gene expression patterns
+ - DPT-based analysis proceeds by identifying branching points that occur after cells progress from a root cell through a common ‘trunk trajectory’   
+ <sub>[L. Haghverdi et al. (2016)](http://dx.doi.org/10.1038/nmeth.3971)</sub>
```

Comment:
citations do work in slides, if you want to use those instead. Just add a `slides.bib` file with the appropriate references and use the `{% cite .. %}` command just like you would in the tutorial. It'll render as a link going directly to the DOI.

Reply:
Good to know, will keep that in mind for the future!